io.sourceforge.tartube.Tartube.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.sourceforge.tartube.Tartube</id>
```

Comment:
The website is `tartube.sourceforge.io` so this should be the app-id

```suggestion
  <id>io.sourceforge.tartube</id>
```

And this is confirmed in upstream source code

https://github.com/axcore/tartube/blob/master/tartube/tartube#L83